knowledge_repo/app/templates/base.html

```diff
@@ -114,7 +114,7 @@
         name: 'knowledge_posts',
         limit: 10,
         display: function (item) {
-          return item;
+          return item.title;
```

Comment:
added - just as an aside though, most of the titles are longer than the width of the searchbox anyway, so the author won't show up.